### PR TITLE
Updated Material container transition

### LINKED ISSUE #8504: Update Android page transition animation on 'Platform-specific behaviors and adaptations' page
### Page URL

https://docs.flutter.dev/resources/platform-adaptations#navigation-transitions

### Page source

_No response_

### Describe the problem

It seems that the gif used for the Android page transition is out of date with the latest Material 3 page transition. It seems like this page transition has been updated for Material widgets but its unclear to me if this is the default transition for all Flutter Android apps. 

See Material 3 page transition here: https://m3.material.io/styles/motion/transitions/applying-transitions#bdfb2e9b-8d12-43ce-a597-2af2a0f6c635

### Expected fix

Record a new gif

### Additional context

cc: @mariam

### LINKED ISSUE #8759: Replace mention of Android P, Q transitions with `ZoomPageTransitionsBuilder` on 'Platform-specific behaviors and adaptations' page
### Page URL

https://docs.flutter.dev/resources/platform-adaptations?linkId=68382499#platform-specific-transition-details

### Page source

https://github.com/flutter/website/blob/main/src/resources/platform-adaptations.md

### Describe the problem

On Windows, Linux, and Android, Flutter now defaults to `ZoomPageTransitionsBuilder` per this [breaking change](https://docs.flutter.dev/release/breaking-changes/page-transition-replaced-by-ZoomPageTransitionBuilder). The [Platform-specific transition details](https://docs.flutter.dev/resources/platform-adaptations?linkId=68382499#platform-specific-transition-details) page calls out the differences between Android P and Android Q, which are no longer in use.

### Expected fix

Update the Android part of [this section](https://docs.flutter.dev/resources/platform-adaptations?linkId=68382499#platform-specific-transition-details) to explain `ZoomPageTransitionsBuilder` instead.

### Additional context

This supercedes https://github.com/flutter/website/issues/3629.

The transition is shown on https://m3.material.io/styles/motion/transitions/applying-transitions#bdfb2e9b-8d12-43ce-a597-2af2a0f6c635.

Do with https://github.com/flutter/website/issues/8504

### PR DESCRIPTION
**Pull request by atsansone**
Replaces the transition animations with a zoom-in, zoom-out animation.

Fixes #8759
Fixes #8504

Staged: https://tony-flutter-site--fix-8759-5avd4gju.web.app/platform-integration/platform-adaptations#navigation-transitions

### CODE REVIEW

**Review by HansMuller — APPROVED** [review]
Nice animation! LGTM

File: src/platform-integration/platform-adaptations.md [hunk 1]
```diff
@@ -89,14 +89,7 @@ On **iOS**:
...
 
-On **Android**,
-two page transition animation styles exist depending
-on your OS version:
-
-* Pre API 28 uses a bottom-up animation that
-  [slides up and fades in][].
-* On API 28 and later, the bottom-up animation
-  [slides and clip-reveals up][].
+On **Android**, Flutter uses the [`ZoomPageTransitionsBuilder`][] animation.
```
Comment (sfshaza2) [inline src/platform-integration/platform-adaptations.md:92]: What does that transition do? Can you explain very briefly, like the original text?

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Please rename this page to match the sidebar.

I wish you would put a very brief description of what this PR does when you create the PR. Having to read open issues is frustrating. For example, I'm not sure why you added/modified all that HTML. I don't see tabs in the generated page. What is it for?

I know that Xiao added all those GIFs and that you didn't do it, but geez, it makes me jumpy and nervous.

File: src/platform-integration/platform-adaptations.md (hunk 1 above)
Comment (atsansone) [reply src/platform-integration/platform-adaptations.md:91]: Done.

**Comment by atsansone** [comment]
@sfshaza2 : Updated the PR summary, changed the title, updated the animation description.

The HTML just matches the existing structures for the page UX (table plus image with captions). Nothing new was done.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**